api/libproject/devstart.rb

```diff
@@ -578,6 +578,33 @@ def cloudsql_import(cmd_name, *args)
                             :fn => lambda { |*args| cloudsql_import("cloudsql-import", *args) }
                         })
 
+def local_mysql_import(cmd_name, *args)
```

Comment:
Can we replace the current big data migrations with this?

Reply:
Yeah. One ui gap we still have is setting the local database to use the db we just imported. How is that handled? Do we automatically update the workbench.cdr_version table ? Do we make it an option --use-for-dev=yes on just use it for dev ? Otherwise why would they download it..